sites/friday/src/create-apps/app-reference/composable-image.md

```diff
@@ -0,0 +1,1352 @@
+| `crons`            | A [cron dictionary](#crons)                         |          | No               | Scheduled tasks for the app.                                                                                                                                                                                                                                     |
+| `source`           | A [source dictionary](#source)                      |          | No               | Information on the app's source code and operations that can be run on it.                                                                                                                                                                                       |
+| `additional_hosts` | An [additional hosts dictionary](#additional-hosts) |          | Yes              | Maps of hostnames to IP addresses.                                                                                                                                                                                                                               |
+
+{{% note %}}
+Please note that, even if available in [Application reference](/create-apps/app-reference/builtin-image.md) when using the built-in image
+of defining images to use in your application container, the  ``type``, ``build``, ``dependencies``, and ``runtime`` keywords
+are not supported if you choose to use Composable Image (`stack`).
+{{% /note %}}
+
+## Root directory
+
+{{% note title="TODO REMOVE ?" %}}
+Does this section can't be removed from here as the [source](#source) exists at the end ?
+{{% /note %}}
+
+Some of the properties you can define are relative to your app's root directory.
+The root defaults to the root of the repository.
+
+```yaml {configFile="app"}
+applications:
+  frontend:
+    stack: [ "nodejs:{{% latest nodejs %}}" ]
+    # Default behavior of source.root
+    source:
+      root: "/"
+    # Additional frontend configuration
+```
+
+That is, if a custom value for `source.root` is not provided in your configuration, the default behavior is equivalent
+to the above.
+
+To specify another directory, for example for a [multi-app project](../multi-app/_index.md),
+use the [`source.root` property](#source).
+
+## Stack
+
+The ``stack`` keyword is used for defining runtimes and other binaries (like `wkhtmltopdf`, or `imagemagick`, see list
+below),
+as a YAML array, that you would like to install in your application container.
+The version is the major (`X`) and sometimes minor (`X.Y`) version numbers,
+depending on the service, as in the following table.
+Security and other patches are taken care of for you automatically.
```

Comment:
This is only true for the runtimes.
As for the other packages, it's up to the Nix maintainers.